iree/compiler/Dialect/HAL/IR/LoweringConfig.td

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef IREE_CONVERSION_COMMON_LOWERINGCONFIG
+#define IREE_CONVERSION_COMMON_LOWERINGCONFIG
+
+// Putting this in HAL dialect for now.
```

Comment:
IREE dialect might be a better place IMO.

Reply:
Ben suggested putting this in HAL dialect.